ypkg2/packages.py

```diff
@@ -210,6 +210,9 @@ def __init__(self, spec):
         # Vala..
         self.add_pattern("/usr/share/vala*/vapi/*", "devel")
 
+        # GIR bindings for gobject
+        self.add_pattern("/usr/share/gir-1.0/*.gir", "devel")
```

Comment:
nit: possibly glob `-1.0` incase this gets bumped in the future

Reply:
It's been `-1.0` for at least 8 years now and if they do change the format/path I'd rather we look into it explicitly.